index.html

```diff
@@ -0,0 +1,71 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <meta http-equiv="X-UA-Compatible" content="ie=edge">
+    <script src="https://kit.fontawesome.com/7b47055909.js" crossorigin="anonymous"></script>
+    <link rel="stylesheet" href="./style.css" type="text/css">
+    <link href="https://fonts.googleapis.com/css?family=Source+Sans+Pro:300&display=swap" rel="stylesheet">
+    <link href="https://fonts.googleapis.com/css?family=Raleway:200&display=swap" rel="stylesheet">
+    <title>Document</title>
+</head>
+<body>
+<!-- <div class="container"> -->
+    <div class="back-ground-image">
+        <div class="nav">
+        
+        <div  class="nav-tabs"><a href="#"><i id="apple" class="fab fa-apple fa-2x"></i></a></div>
+        <div class="nav-tabs"><a href="">Store</a></div>
+        <div class="nav-tabs"><a href="">Mac</a></div>
+        <div class="nav-tabs"><a href="">iPod</a></div>
+        <div class="nav-tabs"><a href="">iPhone</a></div>
+        <div class="nav-tabs"><a href="">iPad</a></div>
+        <div class="nav-tabs"><a href="">iTunes</a></div>
+        <div class="nav-tabs"><a href="">Support</a></div>
+        <!-- <div class="search"> -->

```

Comment:
As can be seen from this [screenshot](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/general/no_inline_comments.png) from [pathwright](https://microverse.pathwright.com/library/fast-track-curriculum/69047/path/step/66243643/)
It is better to delete commented code. If you need to review a previous version of your code, use git history.